
List of Poster and Hands-On session Presenters and their Titles


| Poster No.|	Hands-On No.	|Full name	| Title |
|--|--|--|	 	 
| 	|1	|Rene  Ranzinger	|GRITS Toolbox		|		
|1	| 	|Natasha T Wood	|Molecular dynamics simulations of two glycosylated HIV-1 gp160 trimers with different neutralisation profiles		|		
|2	| 	|Dayoung  Park	|Reorganization of cell surface glycans during malignancy				|
|3	| 	|Masae  Hosoda	|Analysis and development MCAW tool for elucidating recognized glycan patterns by glycan binding proteins		|		
|4	| 	|Keiko  Tokunaga	|GRAB: a visualization tool for displaying glycan associations in RDF				|
|5	| 	|Renji  Inomoto	|Development of a new version of DrawRINGS, a mobile device-supported glycan drawing tool				|
|6	| 	|Shinichi  Higashimoto	|Supporting smart device usage of RINGS glycan analysis tools|
|7	| 	|Yuki  Ushioda	|FlyGlycoDB, the Semantic Web Database of Glycan Related Drosophila Genes				|
| 	|2	|Nobuyuki Paul Aoki	|GlyTouCan: Automated Registration				|
|8	| 	|Daisuke  Shinmachi	|The GlyTouCan Enrichment Cycle: Another Benefit of Registering				|
|9	| 	|Elena  Solovieva	|PAConto - RDF representation of PACDB data and Ontology of Infectious Diseases known to be related to Glycan Binding	|			
|10	| 	|Yushi  Takahashi	|A new glycan multiple alignment algorithm using tree edit distance				|
|11	| 	|Shinichiro  Tsuchiya	|The extension of visualizing utility in the GlycanBuilder for WURCS		|		
|12	| 	|Masaaki  Matsubara	|WURCSFramework – Glycoinformatics Library for Glycan Structure Analysis and Representation using WURCS				|
|13	|3	| Issaku Yamada	A. |GlyTouCan Partner~ Support Site for Glycoscience Research ~ B. GlycoNAVI: Support Tool for Carbohydrate Research |
|14	 |	|Yuki  Kimata	|Simulation Analysis of N-glycan Biosynthesis in Neurospora				|
|15	| 	|Lan  Wang	|E3 ubiquitin ligase CHIP interacts with C-type lectin-like receptor CLEC-2 and promotes its ubiquitin-proteasome degradation|
|16	| 	|David Lloyd Matten	|Automated N-linked glycosylation pipeline				|
|17	 |	|Akihiro  Fujita	|Development of SSKIT:a tool to find key atomic components inglycans				|
|18	| 	|Chen  Zhao	|GlycoStore: a semantic resource of LC-MS data on carbohydrates				|
|19	| 	|Ming-Yi  Ho	|Characterization of unusual sulfated Lewis structure on glycosphingolipid from human colon cancer tissues		|		
|20	| 	|Chia-wei  Lin	|MS-based quantification method for monitoring N-glycosylation process in the cells: SILAC coupled to nanoUPLC-PRM mass spectrometry	|			
|21	| 	|Shinichi  Hirakawa	|Development of a web service for glycan format conversion				|
|22	| 	|Cancelled	 	 |			|
|23	| 	|Yen-Ying  Chen	|One-fraction nanoLC-MS2/MS3 analysis for high throughput glycome-wide precision mapping of glycotopes				|
|24	| 	|Hiroto  Kawashima	|Detection and functional characterization of sialyl Lewis X antigen in mice using novel anti-carbohydrate monoclonal antibodies|
| 	|4	|Marcin Jakub Domagalski	|PepSweetener: a web-based tool to support manual annotation of intact N-linked glycopeptide MS spectra			|	
|25	| 	|Davide  Alocci	|Glycomics@ExPASy: Under the hood|
| 	|5	|Julien  Mariethoz	|Glycomics@ExPASy “Hands on”|
|26	| 	|Toshihiko  Katoh	|Generation of Endo-M mutant that acts on core-fucosylated N- glycans				|
|27	| 	|Jin-Dong  Kim	|GlycoTM – a Collection of Text Mining Resources for GlycoBiology				|
|28	| 	|Barbara  Adamczyk	|Glycomic analysis of paired gastric cancer and non-tumorigenic tissues		|		
|29	| 	|Kazuhiro Aoki	|Comprehensive and functional glycosphingolipidomics by NSI-MS				|
|30	| 	|Varvara  Vitiazeva	|Analysis of highly sulfated O-linked oligosaccharides from serous ovarian cyst fluids			|	
|31	| 	|Miguel Angel Rojas-Macias	|UniCarb-DB: an update|
|32	| 	|Simone  Kurz	|LC-NSI-MS of permethylated glycans|				
|33	| 	|Yukie  Akune	|Comprehensive analysis of the N-glycan biosynthetic pathway using bioinformatics: a theoretical N-glycan structure database	|			
|34	| 	|Barbara  Eckmair	|Multi-step fractionation and mass spectrometry reveal zwitterionic and anionic modifications of the N-glycans of a marine snail|				
|35	| 	|Toshihide  SHIKANAI	|Semantic web application of JCGGDB|
|36	| 	|Takashi  Sato	|Wisteria Lectin Engineering for Specific Detection of Glyco-biomarkers				|
|37	| 	|Shigeko  Kijimoto-OCHIAI	|Existence of CD5 specific sialidase on the mouse thymus cells 			|	
|38	| 	|Yasuhiko  Kizuka	|Chemical biology-based highly sensitive detection of fucosylated glycans with a novel probe			|	
|39	|hands-on	|Melody  Porterfield	|GRITS Toolbox- Storing, processing, and interpreting glycomics data|
| 	|6	|Robert J. Woods	|Gly-Spec: Glycan 3D Structure and Specificity Prediction				|
|40	| 	|Ian Thomas Walsh	|An informatics pipeline to annotate and store quantitative glycan data	|			
|41	| 	|Michiko  Ehara	|Glycan profiling of oral precancerous lesions using a lectin microarray|
|42	| 	|Akira  Togayachi	|Discovering the glycobiomarker for the small cell lung cancer using glycoproteomics techniques			|	
|43	| 	|Kiyohiko  Angata	|Use of whole transcriptome analysis to determine glycogene expression in hepatic cells				|
|44	| 	|Erika  Noro	|Large-scale identification of glycoproteins carrying Lewis x and site-specific glycan alterations in Fut9 knockout mice	|			
|45	| 	|Frederique  Lisacek	|Glycoforest – Distilling Glycomic MS/MS Data				|

